showcase-docs/monitoring-and-logging.md

```diff
@@ -2,20 +2,20 @@
 
 The Backstage Showcase provides a `/metrics` endpoint that provides Prometheus metrics about your backstage application. This endpoint can be used to monitor your backstage instance using Prometheus and Grafana.
 
-When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/janus-idp/helm-backstage), monitoring and logging for your Janus instance can be configured using the following steps.
+When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your Janus instance can be configured using the following steps.
```

Comment:
Are we still keeping the name Janus?

Reply:
nope, good catch. Pushed a few changes related to switching to RHDH name.